source/whats-new.txt

```diff
@@ -82,7 +82,12 @@ To learn more, see the `v4.12.0 Release Highlights <https://github.com/mongodb/n
 What's New in 4.11
 ------------------
 
-The 4.11 {+driver-short+} release includes added support for **mutually
+When using the 4.11 {+driver-short+} with MongoDB Server version 6.0 or
+greater, note that the driver prioritizes ``electionId`` before
+``setVersion`` during Server Discovery and Monitoring events. In
```

Comment:
Suggestion:
While some readers could infer what "prioritizes X over Y" refers to, I think it's easier to understand when identifying that it's the setting value. Maybe something like this would identify the setting values more precisely:
"the driver prioritizes any value in the electionId setting before it considers values in the setVersion setting."